cypress/integration/linkEmbedder.spec.ts

```diff
@@ -0,0 +1,51 @@
+/*
+ * SPDX-FileCopyrightText: 2021 The HedgeDoc developers (see AUTHORS file)
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+
+describe('links to embeddable content', () => {
+  beforeEach(() => {
+    cy.visitTestEditor()
+  })
+
+  it('GitHub Gist', () => {
+    cy.codemirrorFill('https://gist.github.com/schacon/1')
+    cy.getMarkdownBody()
+      .find('span.one-click-embedding.gist-frame')
+      .click()
+    cy.getMarkdownBody()
+      .find('iframe')
+      .should('have.attr', 'title', 'gist schacon/1')
```

Comment:
Why do you rely on the title and not on the explicit URL you set above?

Reply:
Because the URL is a dynamically generated base64 url of our iframe html code.

Reply:
Well. fair. But then I would rather use a data-cy attribute. The title can change at any time.

Reply:
Uses data-cy attribute now